w07h02/test/pgdp/LightPanelTest.java

```diff
@@ -0,0 +1,503 @@
+package pgdp;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import pgdp.security.LightPanel;
+import pgdp.security.SignalPost;
+
+
+class LightPanelTest {
+    private SignalPost testclass;
+
+    @BeforeEach
+    void setUp() {
+        testclass = new LightPanel(0);
+    }
+
+
+    @Test
+    void up() {
+    }
+
+
+    @Test
+    @DisplayName("Level 0")
+    void testLevel0() {
+        assertEqualColorAndLevel("", 0, testclass);
+
+        Assertions.assertTrue(testclass.up("green"));
+        assertEqualColorAndLevel("green", 1, testclass);
```

Comment:
It would be better if you used [static imports](https://en.wikipedia.org/wiki/Static_import#:~:text=Static%20import%20is%20a%20feature,the%20field%20has%20been%20defined.) for assertions?.